CMakeLists.txt

```
@@ -967,6 +969,10 @@ if(IREE_BUILD_EXPERIMENTAL_WEB_SAMPLES)
   add_subdirectory(experimental/web)
 endif()
 
+if (IREE_BUILD_EXPERIMENTAL_CPU_UKERNEL_PLUGINS)
+add_subdirectory(experimental/cpu_ukernel)
+endif()
```

```suggestion
if(IREE_BUILD_EXPERIMENTAL_CPU_UKERNEL_PLUGINS)
  add_subdirectory(experimental/cpu_ukernel)
endif()
```
(match style in file)